public_html/wp-content/mu-plugins/load-other-mu-plugins.php

```diff
@@ -4,6 +4,7 @@
 
 wcorg_include_individual_mu_plugins();
 wcorg_include_mu_plugin_folders();
+wcorg_include_events_network_mu_plugins();
```

Comment:
This function is the main focus of this file, while other changes come from PHPCS auto fix.